storage/src/adb/store/mod.rs

```diff
@@ -184,11 +187,13 @@ pub trait Db<K: Array, V: Codec> {
     fn delete(&mut self, key: K) -> impl Future<Output = Result<bool, Error>>;
 
     /// Commit any pending operations to the database, ensuring their durability upon return from
-    /// this function. Also raises the inactivity floor according to the schedule.
+    /// this function. Also raises the inactivity floor according to the schedule. Returns the
+    /// starting location of the committed range of operations. (The end of the range will be
+    /// `op_count`.)
```

Comment:
We might consider returning a pair of values representing the range of committed operations, where the second value is always op_count().